Поправить якорные ссылки в README.md (#79)

birthdaysgift opened the pull request:
После того как мы добавили содержание в README.md (#27) оказалось, что ссылки, которые мы указали в содержании, не работают в гитхабе. Т.к. мы тестировали эту функциональность только локально и не тестировали в гитхабе, эта особенность осталась незамеченной.

В рамках этой задачи необходимо поправить якорные ссылки в README.md так, чтобы переход от содержания к разделу и от раздела к содержанию работал как локально, так и на гитхабе.

Rey-Lane commented:
Зато теперь они не работают локально в VSC 😅
Хотя в PyCharm работают 🫠
Надо в VSC поставить какой-то другой плагин для `.md` файлов, видимо

birthdaysgift commented:
> Надо в VSC поставить какой-то другой плагин для .md файлов, видимо

Да, у меня стоит [этот](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-preview-github-styles) плагин, в нём норм работает. Но вообще конечно чтобы решение работало вообще везде, со всеми плагинами и во всех IDE нужно просто вынести ссылку "Go back to ToC" из хэдэров. В таком случае всё будет работать как часы, но вот правда выглядеть будет некрасиво :(

birthdaysgift committed:
Поправить якорные ссылки в README.md (#79)

После того как мы добавили содержание в README.md (#27) оказалось, что ссылки, которые мы указали в содержании, не работают в гитхабе. Т.к. мы тестировали эту функциональность только локально и не тестировали в гитхабе, эта особенность осталась незамеченной.

В рамках этой задачи необходимо поправить якорные ссылки в README.md так, чтобы переход от содержания к разделу и от раздела к содержанию работал как локально, так и на гитхабе.
